app/redux/actions/todos.js

```diff
@@ -10,4 +10,9 @@ const completeTodo = payload => ({
   payload
 })
 
-export { addTodo, completeTodo }
\ No newline at end of file
+const deleteTodo = payload => ({
+  type: DELETE_TODO,
+  payload
+})
+
+export { addTodo, completeTodo, deleteTodo }
```

Comment:
Can you add a newline at the end of this file? It will make future Git diffs cleaner.

(ditto throughout)